cluster-autoscaler/core/scale_up.go

```diff
@@ -22,18 +22,16 @@ import (
 
 	apiv1 "k8s.io/api/core/v1"
 	extensionsv1 "k8s.io/api/extensions/v1beta1"
-	"k8s.io/apimachinery/pkg/api/resource"
 	"k8s.io/autoscaler/cluster-autoscaler/cloudprovider"
 	"k8s.io/autoscaler/cluster-autoscaler/clusterstate"
 	"k8s.io/autoscaler/cluster-autoscaler/context"
 	"k8s.io/autoscaler/cluster-autoscaler/estimator"
 	"k8s.io/autoscaler/cluster-autoscaler/expander"
 	"k8s.io/autoscaler/cluster-autoscaler/metrics"
+	ca_processors "k8s.io/autoscaler/cluster-autoscaler/processors"
```

Comment:
Do we need a package alias here? It would be nice to get consistent naming between different files (autoscaler.go does not alias this import, others do).

Reply:
I like processors as a variable / parameter name. It's very descriptive and better than any other option I came up with. I'll change it if you feel strongly though.

Reply:
I'm fine with ca_processors as long as we keep it consistent - so please update the autoscaler.go import (I think it's the only one not aliased) to have both consistency and nice variable names.

Reply:
done